fele-client-service/cli/fele.js

```diff
@@ -122,5 +122,10 @@ userCommand
         }
         waitForUserInput()
     })
+    //program.parse(process.argv);
```

Comment:
Is the fele user command working , when you removed `program.parse(process.argv)` ?  I think it is needed for the fele to work.